ui/src/logic/utils.ts

```diff
@@ -21,10 +24,5 @@ export const makeBrowserNotification = (bin: Bin) => {
 
 export function isNewNotificationSupported() {
   if (!window.Notification || !Notification.requestPermission) return false;
-  try {
-    new Notification('');
-  } catch (e: any) {
-    if (e.name == 'TypeError') return false;
-  }
```

Comment:
this was the fix from #123 probably should leave in

Reply:
This is what is producing duplicate notifications on desktop. Won't the try catch in the makeBrowserNotification function catch this?